### PR TITLE
Replace all occurrences of OS/X

### PR DESCRIPTION
**Pull request by fingolfin**
The Macintosh operating system underwent many name changes over the years,
from being nameless to "System" to "Mac OS" to "OS X" to "macOS".

But at no point in time was it ever called "OS/X" :-)

### CODE REVIEW

File: docs/configuration/advanced/misc.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Possible (incomplete list of) values are:
 
 - auto        The platform will be auto-detected.
 - xcb         Uses XCB plugin (for X11 environment).
-- cocoa       Used to be run on Mac OS/X.
+- cocoa       Used to be run on Mac macOS.
```
Comment (uspasojevic96) [inline docs/configuration/advanced/misc.md:11]:
```suggestion
- cocoa       Used to be run on macOS.
```

**Review by uspasojevic96 — CHANGES_REQUESTED** [review]
Did a quick review, what I would NOT do is to edit existing changelog, but add new entry for the change of naming

**Comment by christianparpart** [comment]
Many thanks, @fingolfin.

I think I'm used to name it like that from the time it was named like that (just without the slash):

image [[1]](https://en.wikipedia.org/wiki/MacOS)

Not sure were the slash i coming from, in my memories :)

File: docs/configuration/advanced/misc.md (hunk 1 above)
Comment (fingolfin) [reply docs/configuration/advanced/misc.md]: Thanks

**Comment by fingolfin** [comment]
> Did a quick review, what I would NOT do is to edit existing changelog,

But why not? Would you not fix typos in them either?

> but add new entry for the change of naming

OK, if you want a changelog entry for a typo fix, I am happy to add it -- except I have no idea how. I am guessing `metainfo.xml` might be the changelog file (though I am not sure given its name)? But what is the format, where would this be added?

**Comment by christianparpart** [comment]
https://github.com/contour-terminal/contour/blob/cdb2e16d29fff252a3b95977c9dc03fd2f96e2a0/metainfo.xml#L107-L113

Hey. This is going to be the changelog for the next release. You can simply add yourself at the end of it.

Changelogs can of course be typo-fixed, but I remember from other big open source projects that change logs are only about to be changed for present/future entries, not touching past entries. So I think that's a philosophy thing :)

**Comment by christianparpart** [comment]
Hey @fingolfin. Can I help you with anything?

**Comment by fingolfin** [comment]
I don't time to work on this outside of weekends. Anyway, this is a trivial search&replace change. Feel free to finish the PR in whichever way you want it, or to close it / do your own version of it.

**Review by christianparpart — APPROVED** [review]

**Comment by christianparpart** [comment]
I've added the changelog entry after the merge. Many thanks @fingolfin for your contribution and support. :-)

**Pull request merged**